client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout.js

```diff
@@ -516,3 +516,47 @@ function useUpdateCartLocationWhenPaymentMethodChanges(
 		}
 	}, [ activePaymentMethod, updateCartContactDetails ] );
 }
+
+function SubmitButtonHeader() {
+	const translate = useTranslate();
+
+	const scrollToTOS = () => document.getElementById( 'checkout-terms' ).scrollIntoView();
+
+	return (
+		<SubmitButtonHeaderUI>
+			{ translate( 'By continuing, you agree to our {{button}}Terms of Service{{/button}}.', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'By continuing, you agree to our {{tosLink}}Terms of Service{{/tosLink}}.' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/checkout-submit-button)